content/efcore/getting-started/samples/defining-unique-identifiers/composite-key/README.md

```diff
@@ -0,0 +1,8 @@
+# Composite Key Using Fluent API
+
+This code example demonstrates how to use EF Core to set properties as a composite key using Fluent API.
```

Comment:
This code example demonstrates how to configure a composite key in EF Core via Fluent API.